AsyncDisplayKit/ASDisplayNode+Beta.h

```diff
@@ -98,4 +98,9 @@ ASDISPLAYNODE_EXTERN_C_END
  */
 - (BOOL)placeholderShouldPersist;
 
+/**
+ * @abstract Cancels all performing layout transitions. Can be called on any thread.
```

Comment:
In a later change, we should add some additional detail here to clarify what the point of no return is for committing a new layout, And perhaps give a short example of a scenario of that you would want to or need to use this method.